src/business/contract/action-creators.js

```diff
@@ -130,3 +130,67 @@ export const contractRaiseDispute =
     throw new Error(err) // FIXME this error should not throw the execution
   }
 }
+
+export const deployRNG = () => async dispatch => {
```

Comment:
I don't understand why you add this feature. I think these contracts (RNG, pinakion and kleros) should be already deployed and maybe added in env.process.

Reply:
I added the code when I was debugging/trying to get everything deployed and it didn't feel right to delete it incase we want to use it again. You are right it is unused though and probably will not be useful outside of dev